Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -47,6 +50,8 @@ extension ModuleError: FixableError {
             return "the target \(module) cannot have the executable \(dependency) as a dependency"
         case .invalidManifestConfig(let package, let message):
             return "invalid configuration in '\(package)': \(message)"
+        case .cycleDetected(let cycle):
+            return "found cyclic dependency declaration: " + cycle.map{$0.name}.joined(separator: " -> ")
```

Comment:
I'd recommend reporting an extra `+ "-> \(cycle[0].name"` at the end to make it explicit where the cycle is.